NitroxServer-Subnautica/Program.cs

```diff
@@ -37,6 +37,7 @@ private static async Task Main(string[] args)
             AppDomain.CurrentDomain.UnhandledException += CurrentDomainOnUnhandledException;
 
             ConfigureCultureInfo();
+            Log.Info($"Starting NitroxServer V{NitroxEnvironment.Version} for Subnautica");
```

Comment:
```suggestion
            Log.Info($"Starting NitroxServer V{NitroxEnvironment.Version}");
```

Reply:
As this is in `NitroxServer-Subnautica` I saw this fitting. Especially with BZ in mind.